src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (eventt) => {
```

Comment:
The variable `eventt` is likely a typo. It should be named `event` to follow common naming conventions and improve code readability.